storage/backup/backup-cli/src/coordinators/backup.rs

```diff
@@ -353,10 +362,50 @@ impl BackupCompactor {
             state_snapshot_file_compact_factor,
             transaction_file_compact_factor,
             concurrent_downloads,
+            expiration_duration,
         }
     }
 
-    pub async fn run(&self) -> Result<()> {
+    /// Update the existing mapping and return the files to be moved out of metadata folder
+    fn update_existing_file_timestamps(
```

Comment:
takes me a few minutes to comprehend what "existing" refers to, probably  also calling it "update_compaction_timestamps" would do.